cpp/notebooks/opencv4.ipynb

```diff
@@ -1,65 +0,0 @@
-{
```

Comment:
Can you also please change the README.md since there's a link to this deleted ipython notebook?

Reply:
Done